### PR TITLE
Create bruce.json

### PR DESCRIPTION
**Pull request by BruceWangyq**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/bruce.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "bruce is a dev",
+  "repo": "https://github.com/brucewangyq",
+  "owner": {
+    "email": "brucewangyq@gmail.com",
+    "twitter": "brucewangdev",
+    "username": "brucewangyq"
+  },
+  "record": {
+    "TXT": "vc-domain-verify=bruce.is-a.dev,a33081fa59e0ed06cc4b",
```
Comment (WilliamDavidHarrison) [inline domains/bruce.json:10]: Sorry, we don't support TXT verification.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/bruce.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/bruce.json:10]: Also, we don't support Vercel.

Comment (BruceWangyq) [reply domains/bruce.json]: Thank you for your reply, I changed to CNAME.

**Review by wdhdev — CHANGES_REQUESTED** [review]
404?

**Comment by BruceWangyq** [comment]
Could you try it again?

**Review by TheBotlyNoob — APPROVED** [review]
your url leads to a 404 page. maybe you could move https://github.com/BruceWangyq/Bruce-Portfolio to `BruceWangyq/brucewangyq.github.io`

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]
wrong button. see above.

**Comment by TheBotlyNoob** [comment]
I'm closing this because of the large amounts of pull requests. You can make another PR later with the fixes if you want.

**Pull request closed without merging**